Jenkinsfile

```diff
@@ -18,7 +18,7 @@ pipeline {
           HELM_RELEASE = "$PREVIEW_NAMESPACE".toLowerCase()
         }
         steps {
-          dir ('/home/jenkins/go/src/github.com/jenkins-x/ext-jacoco') {
+          dir ('/home/jenkins/go/src/github.com/jenkins-x-apps/jx-app-jacoco') {
```

Comment:
BTW all this is nuts. If you use Go modules, you do not need to check out to a specific directory, so all the `dir` steps could either be omitted or take relative paths. (Even the latter case, something like `make -C charts/jx-app-jacoco` is more straightforward.)

Discussion in https://github.com/jenkins-x/draft-packs/issues/83.

Reply:
yeah at the tome of creating the Jenkinsfiles we didn't use go modules, would be good to switch.

Reply:
@jglick I will create a separate PR for that as part of our improvements.